.changeset/fresh-pans-cry.md

```diff
@@ -0,0 +1,5 @@
+---
+'@cypress-design/icon-registry': minor
```

Comment:
```suggestion
'@cypress-design/icon-registry': minor
'@cypress-design/vue-icon': minor
'@cypress-design/react-icon': minor
```

Reply:
@cbfrance Agreed with this point ☝️ 

We should fix this before releasing to make sure we bump the version of each component.